composer.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "spryker-eco/optivo",
+  "autoload": {
+    "psr-0": {
```

Comment:
You shouldnt use psr0 anymore, even for such projects. Best to switch your templates.